### DISCUSSION TITLE
Basic categorization

### DISCUSSION THREAD

**Discussion by starikcetin** [starikcetin/awesome-upm#23] [Meta]
* Code
* Utility
* Asset
* Framework

**Comment by eelstork** [comment]
Wow. Time flies; sorry for not actioning this stuff sooner, had a lot on my plate recently!

**Comment by eelstork** [comment]
Double sorry; seems this reopened it : (

**Comment by starikcetin** [comment]
Hey no probs, I closed the other issue you opened, not this one.

**Comment by favoyang** [comment]
Though openupm has a more broad [topic concept](https://github.com/openupm/openupm/blob/master/data/topics.yml) (package-to-topics is the one-to-many relationship), but for awesome-upm, keep it simple would be a good start.

**Comment by starikcetin** [comment]
Done in #24

**Comment by starikcetin** [comment]
> Though openupm has a more broad [topic concept](https://github.com/openupm/openupm/blob/master/data/topics.yml) (package-to-topics is the one-to-many relationship), but for awesome-upm, keep it simple would be a good start.

Yep, that is very hard to do on a simple list, so a basic categorization is more fitting here.
